.github/workflows/post-submit.yaml

```diff
@@ -34,7 +34,7 @@ jobs:
       - name: Build and push versioned CSV and images for tags
         if: ${{ github.ref_type == 'tag' }}
         # remove leading 'v' from tag!
-        run: export VERSION=$(echo $GITHUB_REF_NAME | sed 's/v//') && make container-build-and-push-community
+        run: export VERSION=$(echo $GITHUB_REF_NAME | sed 's/v//') && export SKIP_VERIFY_PREVIOUS_VERSION="true" && make container-build-and-push-community
```

Comment:
when you want to skip verification for community releases, why not doing that in the Makefile, instead of having to set new env vars when calling the community target?

Reply:
>when you want to skip verification for community releases, why not doing that in the Makefile, instead of having to set new env vars when calling the community target?

Your intuition is partly correct.
I do want to skip verification for community releases but **only** on CI for the bundle image. Thus, I don't want to skip that in the makefile (in general), as I still want the option of running this target locally without skipping the verification.

Soon when there will be automation for creating the bundle images of communities and submitting the PRs, then we might want to revisit the option of local testing with the verification for PREVIOUS_VERSION.

Reply:
>  for community releases but only on CI for the bundle image

I don't understand. Running `make container-build-and-push-community` with `VERSION` set is a doing a community release, no matter if running locally or CI 🤔

Reply:
I will elaborate on how I understand this issue, but in general, when I used "locally" I meant for prior creation of the PRs submission to the community repos.

The community bundle creation includes some modifications to the bundle CSV where one of them is updating the _replaces_ field for defining from which versions we can update to the version of the CSV. In #108 we introduced this field with a check to the `PREVIOUS_VERSION` variable so that the _replaces_ field would have a proper and meaningful value.

IMHO we need this verification in NMO only before we create the PRs for the community repos* (OKD and OperatorHub), and it can be skipped in other places (e.g., at Github with post-PR for pushing to Quay or pre-PR for PR validity).

In the current usage of CI, we would like to skip CI testing to avoid failing it on the "bad" assignment of the `PREVIOUS_VERSION` variable. This part can be improved so there won't be "bad" assignments. But I would prefer to deal with that afterward.

*Eventually we realized that OperatorHub CI is complaining about this field... So it is only needed for the RH community (/OKD)